source/marsGameHud.vwf.yaml

```diff
@@ -0,0 +1,468 @@
+---
+extends: http://vwf.example.com/hud/overlay.vwf
+implements:
+  - http://vwf.example.com/sceneGetter.vwf
+properties:
+  visible: true
+  lastBlocklyNodeID:
+methods:
+  elementPreDraw:
+    parameters:
+      - context
+      - element
+    body: |
+      var alpha = 1;
+      var time = Date.now() / 1000;
+      if ( !element.enabled ) {
+        alpha = 0.5;
+      } else if ( element.isBlinking ) {
+        if ( time  - element.lastBlinkTime > element.blinkInterval ) {
+          alpha = 0.5;
+          if ( time - element.lastBlinkTime > element.blinkInterval + element.blinkDuration ) {
+            element.lastBlinkTime = time;
+          }
+        }
+      }
+      context.globalAlpha = alpha;
+  elementPostDraw:
+    parameters:
+      - context
+      - element
+    body: |
+      context.globalAlpha = 1;
```

Comment:
Nope. That's just there because the element is being passed in under the covers. It's more for our sake, just to know it's there if we need it. As per my other comment, I need to find a way to make some things more clear to the developer.